packages/xstate-graph/package.json

```diff
@@ -37,12 +37,11 @@
     "xstate": "^4.7.0"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.11",
     "jest": "^24.8.0",
     "lerna-alias": "3.0.3-0",
-    "ts-jest": "^24.0.2",
+    "ts-jest": "^24.1.9",
     "typescript": "^3.7.2",
-    "xstate": "^4.7.0"
+    "xstate": "*"
```

Comment:
Well, I learned something new!

Reply:
😛 this was `manypkg` recommendation - which indeed makes sense, we don't have to worry about version mismatch for local/monorepoed packages in dev dependencies as we always want to use whatever is local and there is no way (at least not currently) for this to affect installations from package manager registries